tests/Fallback/AssetFallbackTest.php

```
@@ -77,124 +39,152 @@ public static function getSaveData(): array
 
-        $this->fs
-            ->expects($this->once())
-            ->method('remove')
-            ->with('package.json')
-            ->willThrowException(new \RuntimeException('Remove failed.'));
+        $this->oldCwd = getcwd();
+        $this->cwd = sys_get_temp_dir() . DIRECTORY_SEPARATOR . uniqid('foxy_asset_fallback_test_', true);
+        $this->config = new Config(['fallback-asset' => true]);
+        $this->io = $this->createMock(IOInterface::class);
+        $this->fs = $this->createMock(Filesystem::class);
+        $this->sfs = new \Symfony\Component\Filesystem\Filesystem();
```

PHPMD (2.15.0)

Missing class import via use statement (line '167', column '26'). (undefined)

(MissingImport)